/husarion_bag_recorder_node:
 ros__parameters:
  lidar_subscriber_topic: livox/lidar
  imu_lidar_subscriber_topic: livox/imu
  vehicle_odom_subscriber_topic: odometry/filtered
  joystick_topic: joy
  file_path: /husarionBag